src/core/test/system/system.t.sol

```diff
@@ -43,18 +44,15 @@ contract SystemTest is TestUtils, DSTest {
         admin = new AdminUser();
         admin_ = address(admin);
         admin.file(borrowerDeployer);
+        juniorInvestor = new WhitelistInvestor(address(lenderDeployer.juniorOperator()), currency_, address(lenderDeployer.juniorERC20()));
+        juniorInvestor_ = address(juniorInvestor);
 
         // give admin access rights to contract
         // root only for this test setup
         rootAdmin.relyBorrowAdmin(admin_);
 
-        // todo replace with investor contract
-        rootAdmin.relyLenderAdmin(address(this));
-
-        // give invest rights to test
-        WhitelistOperator juniorOperator = WhitelistOperator(address(lenderDeployer.juniorOperator()));
-        juniorOperator.relyInvestor(address(this));
-
+        // junior investor
+        rootAdmin.relyLenderAdmin(juniorInvestor_);
```

Comment:
The investor should be an admin.  The investor should be only allowed to call the tranche.